pkgs/by-name/de/devhub/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchurl
+, undmg
+, appimageTools
+}:
+let
+  pname = "devhub";
+  version = "0.102.0";
+  src = fetchurl {
+    x86_64-darwin = {
+      url = "https://github.com/devhubapp/devhub/releases/download/v${version}/DevHub-${version}.dmg";
+      hash = "sha256-1vNI++u9mssvCkVU/6tjVHmrPyEFwKlLbB113c+S1MI=";
+    };
+    x86_64-linux = {
+      url = "https://github.com/devhubapp/devhub/releases/download/v${version}/DevHub-${version}.AppImage";
+      hash = "sha256-KGIEM8+dNhUeIXhu1nuo78XA+EUXvLrYeaMHk+VY0sY=";
+    };
+  }.${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
+
+  meta = {
+    homepage = "https://devhubapp.com/";
+    description = "Desktop app to help you manage GitHub Notifications";
+    license = lib.licenses.gpl3Only;
+    maintainers = with lib.maintainers; [ ByteSudoer ];
+    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
+  };
```

Maybe we could use the sources as the list of supported platforms?

```suggestion
  sources = {
    x86_64-darwin = {
      url = "https://github.com/devhubapp/devhub/releases/download/v${version}/DevHub-${version}.dmg";
      hash = "sha256-1vNI++u9mssvCkVU/6tjVHmrPyEFwKlLbB113c+S1MI=";
    };
    x86_64-linux = {
      url = "https://github.com/devhubapp/devhub/releases/download/v${version}/DevHub-${version}.AppImage";
      hash = "sha256-KGIEM8+dNhUeIXhu1nuo78XA+EUXvLrYeaMHk+VY0sY=";
    };
  };
  
  src = fetchurl (sources.${stdenv.system} or (throw "Unsupported system: ${stdenv.system}"));

  meta = {
    homepage = "https://devhubapp.com/";
    description = "Desktop app to help you manage GitHub Notifications";
    license = lib.licenses.gpl3Only;
    maintainers = with lib.maintainers; [ ByteSudoer ];
    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
    platforms = lib.attrNames sources;
  };
```